src/test/game/BoardTest.java

```diff
@@ -24,32 +25,46 @@ public void canMakeMoveOnABoard() {
     @Test
     public void canReturnRows() {
         Board board = new Board(emptyBoard);
-        List<List<String>> rows = board.getRows();
-
-        assertEquals(asList("1", "2", "3"), rows.get(0));
-        assertEquals(asList("4", "5", "6"), rows.get(1));
-        assertEquals(asList("7", "8", "9"), rows.get(2));
+        List<Line> rows = board.getRowLines();
+        assertEquals(new Line("1", "2", "3"), rows.get(0));
+        assertEquals(new Line("4", "5", "6"), rows.get(1));
+        assertEquals(new Line("7", "8", "9"), rows.get(2));
     }
 
+
     @Test
     public void canReturnColumns() {
         Board board = new Board(emptyBoard);
-        List<List<String>> rows = board.getColumns();
+        List<Line> rows = board.getColumnLines();
 
-        assertEquals(asList("1", "4", "7"), rows.get(0));
-        assertEquals(asList("2", "5", "8"), rows.get(1));
-        assertEquals(asList("3", "6", "9"), rows.get(2));
+        assertEquals(new Line("1", "4", "7"), rows.get(0));
```

Comment:
I would suggest extracting a helper method in the tests to create a line. If you decide to rename it then you will only have one place to change it.